src/Tempest/Console/src/Commands/Generators/MakeControllerCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Commands\Generators;
+
+use Tempest\Console\ConsoleArgument;
+use Tempest\Console\ConsoleCommand;
+use Tempest\Console\Stubs\ControllerStub;
+use Tempest\Generation\HasGeneratorCommand;
+
+final class MakeControllerCommand
+{
+    use HasGeneratorCommand;
+
+    #[ConsoleCommand(
+        name       : 'make:controller',
+        description: 'Create a new controller class with a basic route',
+        aliases    : ['controller:make', 'controller:create', 'create:controller'],
+    )]
```

Comment:
```suggestion
    #[ConsoleCommand(
        name: 'make:controller',
        description: 'Creates a new controller class with a route',
        aliases: ['controller:make', 'controller:create', 'create:controller'],
    )]
```

One of the reasons I wanted an abstraction initially was to avoid repeating descriptions and aliases. It's easy to become inconsistent about what aliases are available for specific `make` commands.

Reply:
That's true !